mitodl/open-discussions
# Documentation and .env changes for initial setup

#### Pre-Flight checklist

- [ X ] Testing
  - [ X ] Changes have been manually tested

#### What are the relevant tickets?
N/A

#### What's this PR do?
While setting up I found that there was a missing `.env` value and instructions to add it as well as a bad link on the readme. To save someone else the time I lost, I am putting the small doc and `.env.example` changes up for review/merge.

#### How should this be manually tested?
You can check that the link works in the readme and that the value I added to example `.env` file makes sense.  This is to allow the webpack to build. Without adding this value, my image was failing to fully build.

#### Any background context you want to provide?
I already said this, but I lost a good amount of time trying to figure out why this wasn't set as it wasn't super obvious what was feeding $HOST in the docker build. Also had to track down the generic web app setup to the new github.io page :)
